content/docs/command-reference/add.md

```diff
@@ -148,6 +171,20 @@ not.
   > Note that external outputs typically require an external cache setup. See
   > link above for more details.
 
+- `--to-remote` - add target data into DVC and create a .dvc file, but instead
+  of caching it into DVC cache, transfer it straight to remote storage. Check
+  [this](#transferring-data-directly-to-the-remote) section for the details. If
+  this option is specified target can be any cloud or local URL, not necessarily
+  a local file or directory from the workspace as it is required in the regular
+  dvc addworkflow.
+
+- `-o <path>`, `--out <path>` - destination path for the transferred data. (Can
+  only be used with `--to-remote`)
+
+- `-r <name>`, `--remote <name>` - name of the
+  [remote storage](/doc/command-reference/remote). (Can only be used with
+  `--to-remote`)
```

Comment:
```suggestion
- `-r <name>`, `--remote <name>` - name of the
  [remote storage](/doc/command-reference/remote) to transfer external target to
  (can only be used with `--to-remote`).

- `-o <path>`, `--out <path>` - destination path for the transferred data (can
  only be used with `--to-remote`).
```

But the need for `--out` is unclear to me, please see https://github.com/iterative/dvc/pull/5198#pullrequestreview-569646959